### PR TITLE
Python2 code remove from deploy_stub 

### PR DESCRIPTION
**Pull request by Chitresh13**

### CODE REVIEW

File: pandatool/src/deploy-stub/deploy-stub.c [hunk 1]
```diff
@@ -430,62 +430,6 @@ int Py_FrozenMain(int argc, char **argv)
...
-      sys_stream = PySys_GetObject("stderr");
-      if (sys_stream && PyFile_Check(sys_stream)) {
-        PyFile_SetEncodingAndErrors(sys_stream, "mbcs", NULL);
-      }
-    }
-#endif
-
-    if (Py_VerboseFlag)
-        fprintf(stderr, "Python %s\n%s\n",
-            Py_GetVersion(), Py_GetCopyright());
```
Comment (rdb) [inline pandatool/src/deploy-stub/deploy-stub.c]: Any particular reason why you have removed these last three lines, which don't seem to be related to Python 2/3?  Or were these removed by mistake?

Comment (Chitresh13) [reply pandatool/src/deploy-stub/deploy-stub.c]: I thought those lines were under #if condition starting from line 433. So I thought to remove these also.

Comment (rdb) [reply pandatool/src/deploy-stub/deploy-stub.c:488]: The #if condition starting on line 433 ends on line 484.

Comment (Chitresh13) [reply pandatool/src/deploy-stub/deploy-stub.c:488]: Ok! I have corrected the changes.
